docs/pages/management/join-services-to-your-cluster.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Join Services to your Teleport Cluster
+description: How to register the Proxy Service, Database Service, and other Telepoert services with your cluster.
+---
+
+A **Teleport service** manages access to resources in your infrastructure, such
+as Kubernetes clusters, Windows desktops, internal web applications, and
+databases. A **Teleport process** can run multiple Teleport services.
+
+There are multiple methods you can use to join a Teleport process to your
+cluster in order to run Teleport services, including an instance of the Proxy
+Service. Choose the method that best suits your network topology or cloud
+provider:
+
+|Method|Description|When to use|
+|------|-----------|-----------|
+|[EC2 Identity Document](./join-services-to-your-cluster/aws-ec2.mdx)|A Teleport process running on an EC2 instance authenticates to your cluster via a signed EC2 identity instance identity document.|Your Teleport process will run on EC2.|
```

Comment:
There's nothing to do if this is the best / most accurate term, just wanted to call out the phrase "EC2 identity instance identity document" as possibly redundant. Could we purge the first instance of "identity"?